keyboards/idobo/keymaps/ifohancroft/config.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+// place overrides here
+#undef USB_POLLING_INTERVAL_MS
```

Comment:
This shouldn't be set by default, so no need to undefine it
```suggestion
```

Reply:
Ah! Thank you! Not sure why I thought it's defined by default.

Reply:
Done